pkg/daemon/ceph/osd/daemon.go

```diff
@@ -106,47 +104,31 @@ func handleTerminate(context *clusterd.Context, lvPath, volumeGroupName string)
 			if err != nil {
 				return fmt.Errorf("failed to kill ceph-osd process. %+v", err)
 			}
-			if err := releaseLVMDevice(context, volumeGroupName); err != nil {
-				return fmt.Errorf("failed to release device from lvm. %+v", err)
-			}
 			return nil
 		}
 	}
 }
 
 func killCephOSDProcess(context *clusterd.Context, lvPath string) error {
 
-	processKilled := false
 	pid, err := context.Executor.ExecuteCommandWithOutput(false, "", "fuser", "-a", lvPath)
 	if err != nil {
 		return fmt.Errorf("failed to retrieve process ID for - %s. Error %+v", lvPath, err)
 	}
 
-	logger.Debugf("process ID for ceph-osd: %s", pid)
+	logger.Infof("process ID for ceph-osd: %s", pid)
```

Comment:
```suggestion
	logger.Debugf("process ID for ceph-osd: %q", pid)
```

This can be a debug message I think.